examples/src/main/scala/org/apache/spark/examples/streaming/KafkaWordCount.scala

```diff
@@ -27,18 +28,18 @@ import org.apache.spark.streaming.kafka._
 import org.apache.spark.SparkConf
 
 /**
- * Consumes messages from one or more topics in Kafka and does wordcount.
- * Usage: KafkaWordCount <zkQuorum> <group> <topics> <numThreads>
- *   <zkQuorum> is a list of one or more zookeeper servers that make quorum
- *   <group> is the name of kafka consumer group
- *   <topics> is a list of one or more kafka topics to consume from
- *   <numThreads> is the number of threads the kafka consumer should use
- *
- * Example:
- *    `$ bin/run-example \
- *      org.apache.spark.examples.streaming.KafkaWordCount zoo01,zoo02,zoo03 \
- *      my-consumer-group topic1,topic2 1`
- */
+  * Consumes messages from one or more topics in Kafka and does wordcount.
```

Comment:
This comment style is inconsistent with the rest of Spark (also makes it difficult to see what parts of the comment has/hasn't changed).